### ISSUE TITLE
[TI-76] Shortening an interval over an exclusion makes the exclusion no longer apply

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/timewarrior#80]
_A M on 2017-05-09T01:23:40Z says:_ 

 I'm not sure how to write a test for this, sorry!  It appears to me that
shortening an interval which spanned an exclusion makes the exclusion
no longer apply to that interval.  Earlier today I when looking at my day summary the exclusion was applied.  I just noticed I had left the task running and stopped it.  Now the day summary no longer applies the exclusion.

From my shell history earlier today:

```
╰─➤  timew summary :ids                  

Wk  Date       Day ID Tags                                          Start      End    Time   Total
W19 2017-05-08 Mon @4 Seq non int., pcg, pcg-14.1, pr84573, work  8:37:27  9:59:36 1:22:09
                   @3 APT User's, meeting                         9:59:49 10:59:52 1:00:03
                   @2 Seq non int., pcg, pcg-14.1, pr84573, work 11:00:00 12:00:00 1:00:00
                   @1 Seq non int., pcg, pcg-14.1, pr84573, work 13:00:00        - 0:43:39 4:05:51
                                                                                                  
                                                                                           4:05:51
```

Now, after shortening the interval:

```

╰─➤  timew summary :ids

Wk  Date       Day ID Tags                                          Start      End    Time   Total
W19 2017-05-08 Mon @3 Seq non int., pcg, pcg-14.1, pr84573, work  8:37:27  9:59:36 1:22:09
                   @2 APT User's, meeting                         9:59:49 10:59:52 1:00:03
                   @1 Seq non int., pcg, pcg-14.1, pr84573, work 11:00:00 16:03:52 5:03:52 7:26:04
                                                                                                  
                                                                                           7:26:04

╰─➤  cat ~/.timewarrior/timewarrior.cfg 
exclusions.friday    = <7:30 12:00-13:00 >16:30
exclusions.thursday  = <7:30 12:00-13:00 >16:30
exclusions.wednesday = <7:30 12:00-13:00 >16:30
exclusions.tuesday   = <7:30 12:00-13:00 >16:30
exclusions.monday    = <7:30 12:00-13:00 >16:30
exclusions.saturday  = >0:00
exclusions.sunday    = >0:00
```

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2017-05-09T01:23:40Z
Modified: 2017-07-30T11:11:50Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2017-07-30T11:11:50Z says:_ 

 Good data, thank you.

**Comment by lauft** [comment]
It seems it got fixed along with TI-102, already included in 1.1.1
